templates/prometheus

```diff
@@ -16,8 +16,8 @@ type {{$decorator}} struct {
 
 var {{down .Interface.Name}}DurationSummaryVec = promauto.NewSummaryVec(
   prometheus.SummaryOpts{
-    Name: "{{down .Interface.Name}}_duration_seconds",
-    Help: "{{down .Interface.Name}} runtime duration and result",
+    Name: "{{ down (or .Vars.MetricPrefix .Interface.Name) }}_duration_seconds",
+    Help: "{{ down (or .Vars.MetricPrefix .Interface.Name) }} runtime duration and result",
```

Comment:
I think passing full metric name is better option than just passing a prefix since potential user has more control over what is going to prometheus.

Also not sure we have to put prefix into the help message I would just leave the original interface name or even {{.Interface.Type}}

Can you please do it similarly to the $decorator var above? So that is clear from the first few lines what vars can you pass to the template.

Reply:
All done